src/ostree/ot-builtin-summary.c

```diff
@@ -55,10 +109,16 @@ ostree_builtin_summary (int argc, char **argv, GCancellable *cancellable, GError
 
   if (opt_update)
     {
+      g_autoptr(GVariant) additional_metadata = NULL;
+
       if (!ostree_ensure_repo_writable (repo, error))
         goto out;
 
-      if (!ostree_repo_regenerate_summary (repo, NULL, cancellable, error))
+      additional_metadata = argv_to_additional_metadata (&argc, &argv, error);
```

Comment:
A legitimate bikeshed. I went with positional args because they need less typing to use, but would be just as happy with a named arg if you think the positional ones are best saved for something else, or are inconsistent with other commands, or something. Shall I change it?